app/components/avo/fields/stars_field/edit_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= field_wrapper(**field_wrapper_args) do %>
+  <div data-controller="stars-field">
+    <%= @form.hidden_field @field.id,
+            value: @field.value,
+            data: { stars_field_target: "valueInput" } %>
+    
+    <div class="rating-group flex items-center gap-1 mb-3" 
+         data-component="avo/fields/common/stars_component"
+         data-stars-field-target="starsContainer"
+         data-max="<%= @field.max %>"
+         data-current-value="<%= @field.value || 0 %>">
+      <% (1..@field.max).each do |i| %>
+        <% star_classes = "star #{'filled' if i <= (@value || 0)} inline-block mr-0.5" %>
```

Comment:
Let's use the `class_names` helper for better readability, example: https://github.com/avo-hq/avo/blob/4-dev/app/components/avo/row_selector_component.html.erb#L8